rewrite-gradle/src/main/java/org/openrewrite/gradle/search/FindJVMTestSuites.java

```diff
@@ -60,58 +55,31 @@ public String getDescription() {
         boolean tableAvailable = this.insertRows == null || this.insertRows;
-        return Preconditions.check(new IsBuildGradle<>(), new JavaIsoVisitor<ExecutionContext>() {
-            private boolean isJVMTestSuitesBlock() {
-                Cursor parent = getCursor().getParent();
-                if (parent != null) {
-                    Iterator<Object> path = parent.getPath(J.MethodInvocation.class::isInstance);
-                    if (path.hasNext() && "suites".equals(((J.MethodInvocation) path.next()).getSimpleName())) {
-                        return path.hasNext() && "testing".equals(((J.MethodInvocation) path.next()).getSimpleName());
-                    }
-                }
-                return false;
+        return Preconditions.check(new IsBuildGradle<>(), jvmTestSuite().asVisitor((suite, ctx) -> {
+            if (tableAvailable) {
+                jvmTestSuitesDefined.insertRow(ctx, new JVMTestSuitesDefined.Row(suite.getName()));
             }
-
-            private boolean definesDependencies(J.MethodInvocation suite) {
-                for (Expression suiteDefinition : suite.getArguments()) {
-                    if (suiteDefinition instanceof J.Lambda) {
-                        for (Statement statement : ((J.Block) ((J.Lambda) suiteDefinition).getBody()).getStatements()) {
-                            if (statement instanceof J.Return) {
-                                Expression expression = ((J.Return) statement).getExpression();
-                                if (expression instanceof J.MethodInvocation) {
-                                    if ("dependencies".equals(((J.MethodInvocation) expression).getSimpleName())) {
-                                        return true;
-                                    }
-                                }
-                            }
-                        }
-                    }
-                }
-                return false;
-            }
-
-            @Override
-            public J.MethodInvocation visitMethodInvocation(J.MethodInvocation method, ExecutionContext ctx) {
-                if (isJVMTestSuitesBlock()) {
-                    if (tableAvailable) {
-                        jvmTestSuitesDefined.insertRow(ctx, new JVMTestSuitesDefined.Row(method.getSimpleName()));
-                    }
-                    return !requireDependencies || definesDependencies(method) ? SearchResult.found(method) : method;
-                }
-                return super.visitMethodInvocation(method, ctx);
-            }
-        });
+            return SearchResult.found(suite.getTree());
+        }));
     }
 
-    public static Set<String> jvmTestSuiteNames(Tree tree, boolean definesDependencies) {
-        return TreeVisitor.collect(new FindJVMTestSuites(definesDependencies, false).getVisitor(), tree, new HashSet<>())
+    public static Set<String> jvmTestSuiteNames(Tree tree) {
```

Comment:
This is technically no longer used, so we could remove it entirely, but it is a part of the public API surface.